bots/tests-scan

```diff
@@ -21,68 +21,74 @@
-
-DEFAULT_VERIFY = {
-    'avocado/fedora': BRANCHES,
-    'container/kubernetes': [ 'master', 'rhel-7.6', 'rhel-7.7' ],
-    'container/bastion': BRANCHES,
-    'selenium/firefox': BRANCHES,
-    'selenium/chrome': BRANCHES,
-    'verify/centos-7': [ 'master' ],
-    'verify/continuous-atomic': [ ],
-    'verify/debian-stable': [ 'master' ],
-    'verify/debian-testing': [ 'master' ],
-    'verify/fedora-i386': [ ],
-    'verify/fedora-29': [ 'master' ],
-    'verify/fedora-30': [ 'master' ],
-    'verify/fedora-atomic': [ 'master' ],
-    'verify/fedora-testing': [ ],
-    'verify/ubuntu-1804': [ 'master' ],
-    'verify/ubuntu-stable': [ 'master' ],
-}
-
-# Non-public images used for testing
-REDHAT_VERIFY = {
-    "verify/rhel-7-6": [ 'rhel-7.6' ],
-    "verify/rhel-7-6-distropkg": [ 'master' ],
-    "verify/rhel-7-7": [ 'rhel-7.7', 'master' ],
-    "verify/rhel-8-0": [ 'master', 'rhel-8.0', 'rhel-8-appstream' ],
-    "verify/rhel-8-1": [ 'master', 'rhel-8.1', 'rhel-8-appstream' ],
-    "verify/rhel-atomic": [ 'master' ],
-    'selenium/edge': [ 'master' ],
-}
-
-# Test contexts for projects that use Cockpit's bots and/or images
-EXTERNAL_PROJECTS = {
-    'cockpit-project/starter-kit': [
-        'cockpit/centos-7',
-        'cockpit/fedora-29',
-    ],
-    'cockpit-project/cockpit-ostree': [
-        'cockpit/fedora-atomic',
-        'cockpit/continuous-atomic',
-    ],
-    'cockpit-project/cockpit-podman': [
-        'cockpit/fedora-29',
-    ],
-    'weldr/welder-web': [
-        'cockpit/fedora-29/chrome',
-        'cockpit/fedora-29/firefox',
-    ],
+REPO_BRANCH_CONTEXT = {
+    'cockpit-project/cockpit': {
+        'master': ['avocado/fedora', 'container/kubernetes', 'container/bastion',
+            'selenium/firefox', 'selenium/chrome', 'verify/centos-7', 'verify/debian-stable',
+            'verify/debian-testing', 'verify/fedora-29', 'verify/fedora-30', 'verify/fedora-atomic',
+            'verify/ubuntu-1804', 'verify/ubuntu-stable', 'verify/rhel-7-6-distropkg',
+            'verify/rhel-7-7', 'verify/rhel-8-0', 'verify/rhel-atomic', 'selenium/edge',
+            'verify/rhel-8-1',
+        ],
+        'rhel-7.6': ['avocado/fedora', 'container/kubernetes', 'container/bastion',
+            'selenium/firefox', 'selenium/chrome', 'verify/rhel-7-6',
+        ],
+        'rhel-7.7': ['avocado/fedora', 'container/kubernetes', 'container/bastion',
+            'selenium/firefox', 'selenium/chrome', 'verify/rhel-7-7',
+        ],
+        'rhel-8.0': ['avocado/fedora', 'container/bastion', 'selenium/firefox', 'selenium/chrome',
+            'verify/rhel-8-0',
+        ],
+        'rhel-8-appstream': ['avocado/fedora', 'container/bastion', 'selenium/firefox',
+            'selenium/chrome', 'verify/rhel-8-0', 'verify/rhel-8-1',
+        ],
+        'rhel-8.1': ['verify/rhel-8-1',
+        ],
+        '_manual': ['verify/continuous-atomic', 'verify/fedora-i386', 'verify/fedora-testing',
```

Comment:
Can you please add a comment like `# these can be triggered manually with bots/tests-trigger`